examples/sglang/components/embedding_worker.py

```
@@ -0,0 +1,106 @@
+            ModelType.Embedding,
+            endpoint,
+            self.engine_args.model_path,
+            self.engine_args.served_model_name,
+        )
+
+    @endpoint()
+    async def generate(self, request: EmbeddingRequest):
+        if isinstance(request.input, str):
+            input = request.input
+        elif isinstance(request.input, list):
+            input = [i for i in request.input]
```

Pylint (3.3.7)

[refactor] Unnecessary use of a comprehension, use list(request.input) instead.

(R1721)